packages/firebase/index.d.ts

```diff
@@ -4008,7 +4008,35 @@ declare namespace firebase.analytics {
      */
     logEvent(
       eventName: 'add_payment_info',
-      eventParams?: { [key: string]: any },
+      eventParams?: {
+        coupon?: EventParams['coupon'];
+        currency?: EventParams['currency'];
+        items?: EventParams['items'];
+        payment_type?: EventParams['payment_type'];
+        value?: EventParams['value'];
+        [key: string]: any;
+      },
+      options?: firebase.analytics.AnalyticsCallOptions
+    ): void;
+
+    /**
+     * Sends analytics event with given `eventParams`. This method
+     * automatically associates this logged event with this Firebase web
+     * app instance on this device.
+     * List of official event parameters can be found in
+     * {@link https://developers.google.com/gtagjs/reference/event
+     * the gtag.js reference documentation}.
+     */
+    logEvent(
+      eventName: 'add_shipping_info',
```

Comment:
Where are the new events and their associated params defined? I don't see them in the proposal.

Reply:
See spreadsheet linked in internal bug: b/151107871